acceptance-tests/readme.md

```diff
@@ -0,0 +1,100 @@
+# Acceptance Tests
+
+This module contains the acceptance tests for the Bitbucket Server integration plugin, including the UI tests.
+
+## Running the tests
+
+The Jenkins [Acceptance Test Harness framework](https://github.com/jenkinsci/acceptance-test-harness) runs an instance 
+of Jenkins per test and installs the plugin under test using the `.hpi` file in the `../target` folder, defined using 
```

Comment:
Instead of `../target` folder, we should say {project.build.directory} (generally ../target)

Reply:
This one's a bit different, but see my comment about hard-coded `target` in the pom^.